ksqldb-execution/src/main/java/io/confluent/ksql/execution/ddl/commands/CreateTableCommand.java

```diff
@@ -36,7 +37,8 @@ public CreateTableCommand(
       @JsonProperty(value = "topicName", required = true) final String topicName,
       @JsonProperty(value = "formats", required = true) final Formats formats,
       @JsonProperty(value = "windowInfo") final Optional<WindowInfo> windowInfo,
-      @JsonProperty(value = "orReplace", defaultValue = "false") final Optional<Boolean> orReplace
+      @JsonProperty(value = "orReplace", defaultValue = "false") final Optional<Boolean> orReplace,
+      @JsonProperty(value = "type", defaultValue = "NORMAL") final Optional<String> type
```

Comment:
I used a `String` instead of the `CreateTable.Type` because of dependencies. I would have to add ksql-parser deps to the ksql-execution, which I'm not sure about it. So I just converted the type to a string from the `CreateSourceFactory` which calls this class. I then converted back to `CreateTable.Type` on the `DdlCommandExec`.

Reply:
Will use a boolean value instead of a Enum. I had another idea with the types to support types like `external`, `temporary`, etc in the future, so for compatibility I wanted to do it now, but I don't think that's really necessary. If they're supported, then users could mix the types, so we'll end up adding more parameters. I'll switch with the boolean instead.